doc/Development_Documentation/20_Extending_Pimcore/13_Bundle_Developers_Guide/08_Adding_Document_Types.md

```diff
@@ -2,13 +2,11 @@
 
 Previously, the only way to define custom documents was to create them in a special namespace `Pimcore\Model\Document`. This
```

Comment:
IMHO this doc should be more in absolute terms, the relative "Previously" may lose the intended sense and purpose in the long run.
 
```suggestion
Prior 10.6, the only way to define custom documents was to create them in a special namespace `Pimcore\Model\Document`. This
```

Maybe the first line should be something that explain the current state first eg.
```
Defining custom documents can be done in the config via a static mapping from document type to class name.
```
and potentially should move the historical change behind the change into the Upgrade Notes instead.